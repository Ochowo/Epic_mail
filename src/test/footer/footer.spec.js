import React from 'react';
import { shallow } from 'enzyme';
import Footer from '../../components/footer/footer';

const setUp = () => {
  const component = shallow(<Footer />);
  return component;
};
describe('Footer component', () => {
  let component;
  beforeEach(() => {
    component = setUp();
  });
  it('should render without errors', () => {
    expect(component).toMatchSnapshot();
  });
});
