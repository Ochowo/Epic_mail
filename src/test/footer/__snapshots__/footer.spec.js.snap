// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component should render without errors 1`] = `ShallowWrapper {}`;
